pkg/translator/prometheusremotewrite/helper_test.go

```diff
@@ -880,3 +880,87 @@ func TestAddSingleHistogramDataPoint(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateLabels(t *testing.T) {
```

Comment:
Maybe add a test case for defined `extras`, but empty `baseLabels`?

Reply:
Done, PTAL.